clients/client-s3/test/unit/authSchemePreference.spec.ts

```diff
@@ -0,0 +1,48 @@
+import { describe, expect, test as it } from "vitest";
+
+import { S3 } from "../../src/S3";
+import "../../../../packages/signature-v4-crt/src/index";
```

Comment:
```suggestion
import "@aws-sdk/signature-v4-crt";
```

Reply:
Done in [`4fd8203` (#7063)](https://github.com/aws/aws-sdk-js-v3/pull/7063/commits/4fd82035c908eac8078dd2940390dea867d705e3)